src/System.Reflection.Emit.ILGeneration/ref/System.Reflection.Emit.ILGeneration.cs

```diff
@@ -48,7 +48,6 @@ public partial class ILGenerator
         public virtual void Emit(System.Reflection.Emit.OpCode opcode, System.Type cls) { }
         public virtual void EmitCall(System.Reflection.Emit.OpCode opcode, System.Reflection.MethodInfo methodInfo, System.Type[] optionalParameterTypes) { }
         public virtual void EmitCalli(System.Reflection.Emit.OpCode opcode, System.Reflection.CallingConventions callingConvention, System.Type returnType, System.Type[] parameterTypes, System.Type[] optionalParameterTypes) { }
-        public virtual void EmitCalli(System.Reflection.Emit.OpCode opcode, System.Runtime.InteropServices.CallingConvention unmanagedCallConv, Type returnType, Type[] parameterTypes) { }
```

Comment:
Why was this API removed?

Reply:
It was moved into a netcore-specific file since it wasn't present in all frameworks that support netstandard2.0 (eg: nca2.0, uwp)
https://github.com/dotnet/corefx/pull/30741/files/7eb9a5f89446dbb4aa97fa2d3575cca8fe5bae87#diff-17fa736219e2458643a24ea213deb9e4